projects/1.20/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2465 @@
+  "tetra.workbench.module_detail.not_tweakable": "调整不可用",
+  "tetra.workbench.schematic_list.empty": "所选插槽没有可用的升级",
+  "tetra.workbench.schematic_detail.craft": "制作",
+  "tetra.workbench.schematic_detail.back": "返回",
+  "tetra.workbench.schematic_detail.experience": "此操作需要%d级经验",
+  "tetra.workbench.schematic_detail.destabilize_tooltip": "该物品有魔力去稳化的组件，进行此操作有可能对以下组件造成负面效果：",
+  "tetra.workbench.schematic_detail.repair_tooltip": "替换修复循环中的下一个组件，该物品会被§e完全修复",
+  "tetra.workbench.schematic_detail.no_material_tooltip": "该组件需要材料来制作",
+  "tetra.workbench.schematic_detail.material_count_tooltip": "提供的材料不足",
+  "tetra.workbench.schematic_detail.material_tooltip": "提供的材料不能用于制作该组件",
+  "tetra.workbench.schematic_detail.integrity_tooltip": "制作这个会超出物品的完整度上限",
+  "tetra.workbench.schematic_detail.tools_tooltip": "没有此操作所需的工具",
+  "tetra.workbench.schematic_detail.level_tooltip": "制作这个需要更多经验",
+  "tetra.workbench.schematic_list.temporary": "可以制作，因为你的背包中有可用物品，或是附近有工艺图纸",
+  "tetra.sources.multi_tooltip": "此工艺由§9%s§r添加，再由§9%s§r修改或覆写",
+  "tetra.sources.single_tooltip": "此工艺由§9%s§r添加",
+  "tetra.sources.overflow_suffix_label": "，+%s",
+  "tetra.sources.overflow_label": "%s处来源",
+  "tetra.sources.unknown_label": "未知",
+  "tetra.sources.unknown_tooltip": "添加此工艺的模组或数据包未知，可能出现了错误",
+  "tetra.repair_action.label": "修复",
+  "tetra.tool.hammer_dig": "锤",
+  "tetra.tool.hammer_dig.requirement": "此操作需要%d级锤",
+  "tetra.tool.hammer_dig.craft_requirement": "加工需要一个%d级锤",
+  "tetra.tool.hammer_dig.material_requirement": "加工此材料的基础要求是一个%d级锤，此要求可能会因材料的用途不同而不同",
+  "tetra.tool.axe_dig": "斧",
+  "tetra.tool.axe_dig.requirement": "此操作需要%d级斧",
+  "tetra.tool.axe_dig.craft_requirement": "加工需要一个%d级斧",
+  "tetra.tool.axe_dig.material_requirement": "加工此材料的基础要求是一个%d级斧，此要求可能会因材料的用途不同而不同",
+  "tetra.tool.pickaxe_dig": "镐",
+  "tetra.tool.pickaxe_dig.requirement": "此操作需要%d级镐",
+  "tetra.tool.pickaxe_dig.craft_requirement": "加工需要一个%d级镐",
+  "tetra.tool.pickaxe_dig.material_requirement": "加工此材料的基础要求是一个%d级镐，此要求可能会因材料的用途不同而不同",
+  "tetra.tool.shovel_dig": "锹",
+  "tetra.tool.shovel_dig.requirement": "此操作需要%d级锹",
+  "tetra.tool.shovel_dig.craft_requirement": "加工需要一个%d级锹",
+  "tetra.tool.shovel_dig.material_requirement": "加工此材料的基础要求是一个%d级锹，此要求可能会因材料的用途不同而不同",
+  "tetra.tool.hoe_dig": "锄",
+  "tetra.tool.hoe_dig.requirement": "此操作需要%d级锄",
+  "tetra.tool.hoe_dig.craft_requirement": "加工需要一个%d级锄",
+  "tetra.tool.hoe_dig.material_requirement": "加工此材料的基础要求是一个%d级锄，此要求可能会因材料的用途不同而不同",
+  "tetra.tool.cut": "切削",
+  "tetra.tool.cut.requirement": "此操作需要%d级切削工具\n§7大部分的剑可用于切削",
+  "tetra.tool.cut.craft_requirement": "加工需要一个%d级切削工具",
+  "tetra.tool.cut.material_requirement": "加工此材料的基础要求是一个%d级切削工具，此要求可能会因材料的用途不同而不同\n§7大部分的剑可用于切削",
+  "tetra.tool.pry": "撬",
+  "tetra.tool.pry.requirement": "此操作需要%d级撬\n§7羊角头可用于撬",
+  "tetra.tool.pry.craft_requirement": "加工需要一个%d级撬",
+  "tetra.tool.pry.material_requirement": "加工此材料的基础要求是一个%d级撬，此要求可能会因材料的用途不同而不同\n§7羊角头可用于撬",
+  "tetra.tool.available": "你有一个%d级的",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: You have tier %d available**

```suggestion
  "tetra.tool.available": "你有一个%d级的",
```